lib/command/update.rb

```diff
@@ -162,91 +162,93 @@ def execute(argv)
-          data = Downloader.get_data_by_target(target)
-          if !data
-            display_message = "<bold><red>[ERROR]</red></bold> #{target} は管理小説の中に存在しません".termcolor
-          elsif Narou.novel_frozen?(target) && !@options["force"]
-            if argv.length > 0
-              display_message = "ID:#{data["id"]}　#{data["title"]} は凍結中です"
-            else
+      begin
+        update_log = $stdout.capture(quiet: false) do
+          sort_by_key(sort_key, update_target_list).each_with_index do |target, i|
+            display_message = nil
+            data = Downloader.get_data_by_target(target)
+            if !data
+              display_message = "<bold><red>[ERROR]</red></bold> #{target} は管理小説の中に存在しません".termcolor
+            elsif Narou.novel_frozen?(target) && !@options["force"]
+              if argv.length > 0
+                display_message = "ID:#{data["id"]}　#{data["title"]} は凍結中です"
+              else
+                next
+              end
+            end
+            Helper.print_horizontal_rule if i > 0
+            if display_message
+              puts display_message
+              mistook_count += 1
               next
             end
-          end
-          Helper.print_horizontal_rule if i > 0
-          if display_message
-            puts display_message
-            mistook_count += 1
-            next
-          end
-          interval.wait
-          downloader = Downloader.new(target)
-
-          if @options["hotentry"]
-            downloader.on(:newarrival) do |hash|
-              entry = hotentry[hash[:id]] ||= []
-              entry << hash[:subtitle_info]
+            interval.wait
+            downloader = Downloader.new(target)
+
+            if @options["hotentry"]
+              downloader.on(:newarrival) do |hash|
+                entry = hotentry[hash[:id]] ||= []
+                entry << hash[:subtitle_info]
+              end
             end
-          end
 
-          delete_modified_tag = -> do
-            tags = data["tags"] || []
-            data["tags"] = tags - [Narou::MODIFIED_TAG] if tags.include?(Narou::MODIFIED_TAG)
-            data["last_check_date"] = Time.now
-          end
+            delete_modified_tag = -> do
+              tags = data["tags"] || []
+              data["tags"] = tags - [Narou::MODIFIED_TAG] if tags.include?(Narou::MODIFIED_TAG)
+              data["last_check_date"] = Time.now
+            end
 
-          result = downloader.start_download
-          case result.status
-          when :ok
-            delete_modified_tag.call
-            if @options["no-convert"] ||
-                 (@options["convert-only-new-arrival"] && !result.new_arrivals)
-              interval.force_wait
+            result = downloader.start_download
+            case result.status
+            when :ok
+              delete_modified_tag.call
+              if @options["no-convert"] ||
+                   (@options["convert-only-new-arrival"] && !result.new_arrivals)
```

Comment:
__[rubocop]__
* Style/MultilineOperationIndentation: Align the operands of a condition in an `if` statement spanning multiple lines.

<!-- Comment by [SideCI](https://sideci.com) -->